### DISCUSSION TITLE
AirConditioners

### DISCUSSION THREAD

**Discussion by wyc23** [HomeSpan/HomeSpan#289] [Q&A]
is there any tutorial on how to control the airConditioners?

**Comment by TomInIowa** [comment]
Do a search for.  homespan thermostat.    There are some that might work for you.
One that I found "https://github.com/5hubh4m/homespan-thermostat"

**Reply by wyc23** [reply]
thanks for that, but i want to use IR to do a remote control for air conditioner

**Comment by wyc23** [comment]
<img width="357" alt="image" src="https://user-images.githubusercontent.com/99090103/158730751-a0686dc1-5b56-4461-b200-9f00d2153064.png">
i know homespan can work with air conditioner since it's one of the categories on the list, but i don't know how to use it.

**Reply by danilkorotkov** [reply]
https://github.com/unreality/FujiHK
